### ISSUE TITLE
Adjust Food

### ISSUE THREAD

**Issue by elthran** [elthran/UndyingKingdoms#263]
@klondikemarlen - You don't need to code this, but feel free to post ideas here 

Idea 1:
     When your people start starving, excess workers automatically gets switched to 'foraging'. Therefore you start getting some extra food (so shouldn't starve overnight) and when they switch to foragingyou take a -5 happiness penalty.

**Comment by klondikemarlen** [comment]
Perhaps you could drop population growth to 0 (by dropping happiness until pop growth slows) as well, basically just make it stabilize. But not kill all your people and set your happiness to 0. Like what used to happen :P

**Comment by elthran** [comment]
I agree with the first post but not second part. I don't want the game to autoplay itself too much. But I like:

When your people start starving, excess workers automatically gets switched to 'foraging'. Therefore you start getting some extra food (so shouldn't starve overnight) and when they switch to foragingyou take a -5 happiness penalty.
